src/org/thoughtcrime/securesms/components/ScaleStableImageView.java

```diff
@@ -0,0 +1,181 @@
+package org.thoughtcrime.securesms.components;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.util.Log;
+
+import com.bumptech.glide.load.engine.DiskCacheStrategy;
+
+import org.thoughtcrime.securesms.mms.GlideApp;
+import org.thoughtcrime.securesms.util.Util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.widget.AppCompatImageView;
+
+import static android.content.res.Configuration.ORIENTATION_PORTRAIT;
+import static android.content.res.Configuration.ORIENTATION_LANDSCAPE;
+
+/**
+ * An image view that maintains the size of the drawable provided.
+ * That means: when you set an image drawable it will be scaled to fit the screen once.
+ * If you rotate the screen it will be rescaled to fit.
+ * If you crop the screen (e.g. because the soft keyboard is displayed) the image is cropped instead.
+ *
+ * @author Angelo Fuchs
```

Comment:
in general, we do not use the @author tag, this is hard to maintain. the git history should be sufficient.